packages/farms/constants/bsc.ts

```diff
@@ -1325,14 +1325,6 @@ const farms: SerializedFarmConfig[] = [
     token: bscTokens.chr,
     quoteToken: bscTokens.busd,
   },
-  {
-    pid: 96,
-    lpSymbol: '8PAY-BUSD LP',
-    lpAddress: '0x92c3E2cddDb0CE886bCA864151BD4d611A86E563',
-    token: bscTokens['8pay'],
-    quoteToken: bscTokens.busd,
-    isCommunity: true,
-  },
```

Comment:
Don't remove the config, we still have Liquidity in the farm.

Reply:
and what about farm auctions?

Reply:
I don't think that we should remove it.
Just make it an old & new token.